lib/query_builder.ex

```diff
@@ -248,8 +239,8 @@ defmodule QueryBuilder do
             sort_functions: %{},
             changeset: nil
 
-  @spec default_pagination() :: pagination()
-  def default_pagination(), do: @default_pagination
+  alias QueryBuilder.Sort
+  import Sort, only: [is_sort_direction: 1, is_sort_function: 1]
```

Comment:
Let's move those to the top of the file?

Reply:
Done :)